pyro/contrib/gp/kernels/isotropic.py

```diff
@@ -95,26 +96,28 @@ def forward(self, X, Z=None, diag=False):
 
 class SquaredExponential(RBF):
     """
-    SquaredExponential is another name for :class:`RBF` kernel.
+    Another name of :class:`RBF` kernel.
```

Comment:
"another name" -> "alias"

Why are these aliases necessary?

Reply:
In GPML book, the authors use SquaredExponential. But RBF is also a popular name for the same kernel. In scikit-learn, they use ExpSineSquared instead of Periodic. Different papers use different names. So far we have the pairs
+ ExpSineSquared = Periodic
+ SquaredExponential = RBF
+ Matern12 = Exponential
+ Constant = Bias

If we don't want to use aliases, then we can remove them. I have my preferences but willing to hear your opinion. :)